### DISCUSSION TITLE
How to remove the sender name?

### DISCUSSION THREAD

**Discussion by cloudskylimit** [MrMissx/Telegram_Forwarder#15] [Q&A] [answered]
I want to forward some messages without the person who received it will know the sender name, any suggestions please?

**Comment by MrMissx** [comment, marked as answer]
you can change the `forward_message` to `copy_message` with a few modification for the contents

**Reply by cloudskylimit** [reply]
do you mean the `forward_message` in auto_forward.py? and how should be the modification for the contents?

**Reply by MrMissx** [reply]
Yes it is.
You could change it to [telegram.Message.copy](https://python-telegram-bot.readthedocs.io/en/stable/telegram.message.html#telegram.Message.copy) instead of `send_message` I said earlier.
Haven't try it yet but i think that might work and more simple than `send_message`.

**Reply by cloudskylimit** [reply]
wow, it works! you're so great!
thanks Master~ 👍

**Reply by MrMissx** [reply]
No prob... Glad to help you 😊
